lib/elix/responders/lists.ex

```diff
@@ -0,0 +1,105 @@
+defmodule Elix.Responders.Lists do
+  @moduledoc """
+  Commands to create, update, and delete one-dimensional lists and their items.
+  """
+
+  use Hedwig.Responder
+  alias Hedwig.Message
+  alias Elix.Lists
+
+  @usage """
+  show lists - Displays all lists
+  """
+  respond ~r/show lists\Z/i, msg do
+    reply(msg, render_items(Lists.all))
+  end
+
+  @usage """
+  create list <name> - Creates a new list with name
+  """
+  respond ~r/create list (.+)/i, %Message{matches: %{1 => list_name}} = msg do
+    Lists.create(list_name)
+    reply(msg, render_items(Lists.all))
+  end
+
+  @usage """
+  show list <list> - Displays the contents of a list by name or number
+  """
+  respond ~r/show list (.+)/i, %Message{matches: %{1 => list_id}} = msg do
+    list_name = parse_list_identifier(list_id)
```

Comment:
Then again, `String.to_integer/1`, which we're sometimes using anyway, could also do this for us:

```elixir
iex> String.to_integer("9999")
9999

iex> String.to_integer("9999 bottles")
** (ArgumentError) argument error
    :erlang.binary_to_integer("9999 bottles")
```

We'd need to `rescue ArgumentError` and return the string in that case, but also an option...